linter/whitelists/de.txt

```diff
@@ -30,6 +30,7 @@ ausserdem
 Automatisierungsaufgaben
 Avatar
 Avatare
+Backend
```

Comment:
Was that complained by the Linter?

Reply:
Just got ldap and reservation into it. The "Backend" entry seems to have been in our fork for a while. Can't tell if the linter was complaining.